6MI0800226/week1.cpp

```diff
@@ -0,0 +1,114 @@
+#include <iostream>
+#pragma warning(disable: 4996)
+struct transition
+{};
+
+
+//zad 2
+struct pair
+{
+    int firstInt;
+    int secondInt;  
+};
+void inputIntsOfPair(pair& pair1)
+{
+    std::cout << "First integer: ";
```

Comment:
You can always consider typing `using namespace std;` in the beginning. This will get you rid of the burden of writing `std::<whatever>` from there on :P